internal/handlers/common/error.go

```diff
@@ -40,8 +40,8 @@ const (
 
 // Error represents wire protocol error.
 type Error struct {
-	code ErrorCode
 	err  error
+	code ErrorCode
```

Comment:
Nanooptimization (mostly to make a linter happy): make struct smaller due to padding